### PR TITLE
Remove ga

### PR DESCRIPTION
**Pull request by Truhoan**
- Updated build.gradles to no longer reference unnecessary google services.
- Removed code that sends data to google analytics
Issue: ibikecph-android#2 - Remove Google Analytics

### CODE REVIEW

File: ibikecph-app/build.gradle [hunk 1]
```diff
@@ -49,7 +49,7 @@ android {
     }
     buildTypes {
         debug {
-            signingConfig signingConfigs.debug
+            signingConfig signingConfigs.development
```
Comment (emiltin) [inline ibikecph-app/build.gradle]: why the change from debug?

**Review by emiltin — COMMENTED** [review]
it's ok to have debug.keystore in the repo?

**Comment by Truhoan** [comment]
Yes, this is alright. The debug.keystore is generic. 
Developers with Java SDK also have the debug.keystore locally. So I can also remove it from the repo, if you like?

**Comment by emiltin** [comment]
if developers need to modify the content, it needs to to be explained in the readme

**Comment by Truhoan** [comment]
If we keep the debug.keystore in the repo. Developers do not need to modify the content or configure the application with a debug.keystore.

**Comment by emiltin** [comment]
ok. why are you closing this issue?

**Comment by Truhoan** [comment]
Sorry, that was my mistake
